pyglet/shapes.py

```diff
@@ -42,18 +42,28 @@ def on_draw():
 
 .. versionadded:: 1.5.4
 """
+from __future__ import annotations
 
 import math
 
 from abc import ABC, abstractmethod
 
 import pyglet
-
 from pyglet.gl import GL_SRC_ALPHA, GL_ONE_MINUS_SRC_ALPHA
 from pyglet.gl import GL_TRIANGLES, GL_LINES, GL_BLEND
 from pyglet.gl import glBlendFunc, glEnable, glDisable
 from pyglet.graphics import Batch, Group
+from pyglet.graphics.shader import ShaderProgram, ComputeShaderProgram
+
+from typing import (
+    Union,
+    Optional,
+    Tuple,
+    List
+)
```

Comment:
how about just `from typing import Union, Optional, Tuple, List`

Reply:
As imports get longer, it is better to put them on separate line. I do not actually know how many are needed to start putting them on separate lines though.